README.md

```diff
@@ -1,18 +1,23 @@
 # Javascript
+
 ![codequality](https://img.shields.io/lgtm/grade/javascript/github/bellshade/Javascript?label=code%20quality%3A%20js&style=for-the-badge)
 ![codefactor](https://img.shields.io/codefactor/grade/github/bellshade/Javascript/main?label=code%20factor%20quality&style=for-the-badge)
 ![buildTesting](https://img.shields.io/github/workflow/status/bellshade/Javascript/Node%20CI?style=for-the-badge)
 
 Repositori ini berisi kumpulan dari berbagai macam script struktur data dan algoritma yang diimplementasikan dengan menggunakan bahasa pemrograman Javascript.
 
 ## Peraturan dan Pedoman Kontribusi
-Dipersilahkan bila ingin melakukan open pull requets atau kontribusi pada repositori **Javascript**. Sebelum itu, bacalah terlebih dahulu [peraturan dan pedomannya](CONTRIBUTING.md) yang dapat ditemukan pada ``CONTRIBUTING.md``.
+
+Dipersilahkan bila ingin melakukan open pull requets atau kontribusi pada repositori **Javascript**. Sebelum itu, bacalah terlebih dahulu [peraturan dan pedomannya](CONTRIBUTING.md) yang dapat ditemukan pada `CONTRIBUTING.md`.
 
 ## Quick Access
-Semua *script* yang ada pada repositori ini, dapat diakses dengan cepat lewat link berikut:
+
+Semua _script_ yang ada pada repositori ini, dapat diakses dengan cepat lewat link berikut:
+
 - [Backtracking](https://github.com/bellshade/Javascript/tree/main/backtracking)
 - [Cipher](https://github.com/bellshade/Javascript/tree/main/chiper)
 - [Conversion](https://github.com/bellshade/Javascript/tree/main/conversion)
 - [Data Structure](https://github.com/bellshade/Javascript/tree/main/data_structure)
 - [Search](https://github.com/bellshade/Javascript/tree/main/search)
+- [Shorting Algorithm](https://github.com/bellshade/Javascript/tree/main/shorting_algorithm)
```

Comment:
Shorting -> sorting

Reply:
url nya juga wkwk

Reply:
i swear i've changed that for sure. what the hek just happen 🙃🥴

Reply:
ngopi dulu 😂